modules/spring-config/src/main/resources/META-INF/mule.xsd

```diff
@@ -4228,6 +4190,13 @@
                         </xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="ignoreErrorType" type="errorIdentifierType" use="optional">
```

Comment:
no, after the foreach the original collection is the one that continues processing. i checked this